### PR TITLE
modules elbv2 fix

### PR DESCRIPTION
**Pull request by hackacad**
### What does this PR do?
Fix modules/boto_elbv2 if botocore & boto3 are missing

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/42311
https://github.com/saltstack/salt/issues/44062

### Previous Behavior
if python/boto3 and python/botocore are not installed loading the elbv2 module will fail, resulting in the following error message on the minion:

> [ERROR   ] Exception raised when processing __virtual__ function for salt.loaded.int.module.boto_elbv2. Module will not be loaded: 'boto3.assign_funcs'
> [WARNING ] salt.loaded.int.module.boto_elbv2.__virtual__() is wrongly returning `None`. It should either return `True`, `False` or a new name. If you're the developer of the module 'boto_elbv2', please fix this.

### New Behavior
fixes this error

### Tests written?

No

### CODE REVIEW

File: salt/modules/boto_elbv2.py [hunk 1]
```diff
@@ -55,6 +55,8 @@
     # pylint: enable=unused-import
 
     # TODO Version check using salt.utils.versions
+    import boto3
```
Comment (rallytime) [inline salt/modules/boto_elbv2.py]: These imports will need to be moved up into the `unused-import` section with the `import salt.utils.boto3` line:

https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/15590/violations/file/salt/modules/boto_elbv2.py/

**Review by rallytime — CHANGES_REQUESTED** [review]
Thanks @netzmacher - There is a pylint error that needs to be fixed here.

I also think it would be good to add some information to the docstring about the `boto3` dependency. Just something like adding:
```
:depends: boto3
```
Near the top of the docstring.

Would you mind adding that as well to this PR?

**Comment by netzmacher** [comment]
Thanks @rallytime for the help. Just added docstring and change the import.

**Review by rallytime — APPROVED** [review]
Thanks!

**Comment by netzmacher** [comment]
Just noticed that this should fix 
https://github.com/saltstack/salt/issues/42311
&
https://github.com/saltstack/salt/issues/44062

**Pull request merged**